pages/conference/_eventType/index.vue

```diff
@@ -1,51 +1,53 @@
 <template>
-    <div>
-        <i18n-page-wrapper class="pt-24 pb-4" custom-y>
-            <core-h1 :title="title"></core-h1>
-            <p class="intro">{{ intro }}</p>
-        </i18n-page-wrapper>
-        <i18n-page-wrapper
-            v-if="eventType === 'talks'"
-            class="px-4 sm:px-8 md:px-16 lg:px-32 pb-6 lg:pb-12"
-            custom-x
-            custom-y
-        >
-            <p class="filterTitle">Filter</p>
-            <core-hr class="filterHr" custom-margin-y custom-color></core-hr>
-            <checkbox-collection>
-                <core-checkbox
-                    id="ALL"
-                    key="category_ALL"
-                    :value="isAllCategoriesSelected"
-                    :is-checked="isAllCategoriesSelected"
-                    :label="$t('categories.ALL')"
-                    @input="selectAllCategories()"
-                ></core-checkbox>
-                <core-checkbox
-                    v-for="category in speechCategories"
-                    :id="category"
-                    :key="`category_${category}`"
-                    v-model="checkedCategories"
-                    :label="$t(`categories.${category}`)"
-                    :is-checked="isCategoriesChecked(category)"
-                ></core-checkbox>
-            </checkbox-collection>
-        </i18n-page-wrapper>
-        <i18n-page-wrapper
-            class="px-2 sm:px-8 md:px-16 lg:px-32 pb-24"
-            custom-x
-            custom-y
-        >
+    <div class="py-10 md:py-24 lg:py24">
+        <banner>
```

Comment:
The PR for the new banner style (#266) has been merged. You can update the banner on this page after rebasing to the latest main branch.

Reply:
bf38816